defaults/main.yml

```diff
@@ -1,41 +1,32 @@
 ---
 
-php:
-  # List of extra PHP packages to install
-  version: php7.0
-  extras: []
-  pecl: []
-  compiled: []
-  modules:
-    - php7.0-curl
-    - php7.0-intl
-    - php7.0-mcrypt
-    - php7.0-mysql
-    - php7.0-pgsql
-    # E: Unable to locate package php7.0-mssql
-    # - php7.0-mssql
-    - php7.0-sqlite3
-    # removed due to doctrine trying to use memcache or redis by default to cache
-    - php-redis
-    - php-memcache
-  paths:
-    etc: /etc/php7.0
-    fpm_pool: /etc/php7.0/fpm/pool.d
+sansible_php_version: php7.0
+sansible_php_extras: []
+sansible_php_pecl: []
+sansible_php_modules:
```

Comment:
I actually think we should reduce this list right down, I don't think this list make sense to be honest :), let's just install curl by default as pretty much everyone needs that:

```YAML
sansible_php_modules:
  - "{{ sansible_php_version }}-curl"
```